services/web-app/pages/_app.js

```diff
@@ -1,17 +1,16 @@
-/**
+/*
  * Copyright IBM Corp. 2021, 2021
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 import '@/styles/styles.scss'
 
-import Layout, { LayoutProvider } from '@/layouts/layout'
-
-import { DefaultSeo } from 'next-seo'
 import { unstable_FeatureFlags as FeatureFlags } from 'carbon-components-react'
+import { DefaultSeo } from 'next-seo'
+
 import defaultSeo from '@/config/seo.json'
+import Layout, { LayoutProvider } from '@/layouts/layout'
```

Comment:
@mattrosno You're correct that the script only affects the node modules, but one or more of the imports have side effects (which is probably the root of the problem). Changing the `layout` to be imported _after_ carbon components react is what causes the issue to manifest. If I reverse those two lines here, I don't see the problem.

Reply:
Can we `// eslint-disable-next-line simple-import-sort/imports` in that file with side effects until that v11 bug is fixed? (The comment has to be at the start of the file's imports.)

Docs: https://github.com/lydell/eslint-plugin-simple-import-sort/blob/main/examples/ignore.js#L17

Reply:
Yeah, I'll add that to this PR.